src/simularium/SelectionInterface.ts

```diff
@@ -15,9 +18,15 @@ export interface SelectionEntry {
     tags: string[];
 }
 
+export interface ColorChanges {
+    agents: SelectionEntry[];
```

Comment:
this would be better named as agentTypes.

Reply:
on second thought, I realize we use "agents" in other SelectionEntry names as well. so at least it's consistent.   Slightly confusing because we don't set colors on individual agents.